### DISCUSSION TITLE
Ivan Nikolov - PyVista in Medium

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#3126] [Show and tell]
Ivan Nikolov has some amazing work demoing pyvista and a few other libraries for 3D Python visualization:

https://towardsdatascience.com/neighborhood-analysis-kd-trees-and-octrees-for-meshes-and-point-clouds-in-python-19fa96527b77
https://towardsdatascience.com/python-libraries-for-mesh-and-point-cloud-visualization-part-1-daa2af36de30
https://towardsdatascience.com/how-to-voxelize-meshes-and-point-clouds-in-python-ca94d403f81d

**Comment by tkoyama010** [comment]
I just read it. Amazing demoing!!! (ping @IvanNik17 )
